drivers/hmis/app/graphql/types/base_audit_event.rb

```diff
@@ -110,6 +111,11 @@ def graphql_type
       end
     end
 
+    private def custom_assessment_title
+      ca = load_ar_scope(scope: Hmis::Hud::CustomAssessment.with_deleted, id: object.item_id)
```

Comment:
A quick interesting note about using `with_deleted` here -- I only happened to notice this because I've been working recently on the bug ticket about custom assessment deletion, but this doesn't work correctly (won't show the custom assessment's real name, just 'Assessment') when the assessment has been deleted, even though I added `with_deleted` here. The problem is that `FormProcessor` doesn't have `acts_as_paranoid`, so it is really deleted from the DB even when the associated Assessment is just soft-deleted.

Reply:
Ah good find!! That definitely seems problematic because we would not be able to restore an accidentally deleted assessment. I created a [story](https://www.pivotaltracker.com/story/show/187150766) for moving form processor to soft-deletion.

Reply:
Thank you!